ras_party/controllers/account_controller.py

```diff
@@ -975,7 +988,8 @@ def request_survey(survey_id):
 def request_casegroups_for_business(business_id):
     logger.info("Retrieving casegroups for business", business_id=business_id)
     url = f'{current_app.config["CASE_URL"]}/casegroups/partyid/{business_id}'
-    response = Requests.get(url)
+    auth = (current_app.config["SECURITY_USER_NAME"], current_app.config["SECURITY_USER_PASSWORD"])
```

Comment:
why we changed this?

Reply:
It's a long story... We used responses to sort the mocks in our new tests.  It worked in pycharm but for some reason not when you ran it via command line.  We managed to get it working by removing the custom (and silly I think) wrapper around requests and just use it like we do everywhere else.
That then broke a bunch of other tests because I think things were working by luck rather then because proper mocking had been done, after fixing those tests we finally got everything passing!

Basically, it's identical to how it was before and more closely matches how we do http calls in all our other python repos.

Reply:
The previous implementation was using a requests class made by us with less functionality that was also making the tests fail